### ISSUE TITLE
is it possible to set heart rate interval less than 10 min

### ISSUE THREAD

**Issue by sailes-shakya** [Fitpolo/FitpoloDemo_H705_Android#3]
m trying to set the heart rate interval less than 10 min. Can I do it  on changing ZWriteHeartRateIntervalTask class  like

 public ZWriteHeartRateIntervalTask(MokoOrderTaskCallback callback, int heartRateInterval) {
        super(OrderType.WRITE_CHARACTER, OrderEnum.Z_WRITE_HEART_RATE_INTERVAL, callback, OrderTask.RESPONSE_TYPE_WRITE_NO_RESPONSE);
        orderData = new byte[ORDERDATA_LENGTH];
        orderData[0] = (byte) MokoConstants.HEADER_WRITE_SEND;
        orderData[1] = (byte) order.getOrderHeader();
        orderData[2] = (byte) 0x01;
  orderData[3] = (byte) (heartRateInterval );// i remove the * 10 here
    }

**Comment by liuwenzheng** [comment]
Yes,you can do it.It should be noted that shortening the interval between heart rate tests will increase the power consumption of the bracelet.

**Comment by sailes-shakya** [comment]
Then is there any callback function that gives you heart rate at a certain interval?

**Comment by liuwenzheng** [comment]
you should have to read it by the  'ZReadHeartRateTask',no other method
